tests/e2e/main_test.go

```diff
@@ -191,3 +193,53 @@ func getLabelsDocumentation() (map[string][]string, error) {
 	}
 	return documentedMetrics, nil
 }
+
+func TestTelemetryMetrics(t *testing.T) {
```

Comment:
I would rename this to something like "TestKubeStateMetricsErrorMetrics", not exactly that but something close.

Reply:
Ok!